packages/browser-agent-core/src/common/window/session-storage.js

```diff
@@ -19,13 +19,20 @@ const SESS_ID = "NRBA_SESSION_ID"; // prevents potential key collisions in sessi
 function getCurrentSessionIdOrMakeNew() {
   if (!ss) return null; // indicates sessionStoage API not avail
 
-  let sessionId;
-  if ((sessionId = ss.getItem(SESS_ID)) === null) {
-    // Generate a new one if storage is empty (no previous ID was created or currently exists)
-    sessionId = generateRandomHexString(16);
-    ss.setItem(SESS_ID, sessionId);
+  try {
```

Comment:
Per huddle, the problematic line that triggers the DOM exception is actually above where ss is being defined as a side effect:
<img width="747" alt="image" src="https://user-images.githubusercontent.com/28714891/213235282-055ccdb3-edaf-4239-8cbd-98f4f2ee4733.png">

My suggestion is to wrap that instead and leave it as undefined if error thrown (as the getCurrent... function is already setup to handle a null `ss`) like:
```javascript
let ss;
try {
  if (isBrowserScope) ss = window.sessionStorage;
  // else, leave it undefined
} catch (err) {
  // exception thrown, then also leave it undefined
}
```